roles/provision-metrics-apb/tasks/provision-prometheus.yml

```diff
@@ -0,0 +1,129 @@
+- name: Create Prometheus config from template
```

Comment:
@StevenTobin is this a straight copy/paste from what was in the other file?

Reply:
Yes it is, I didn't make any changes.